lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize(menu_name: :italian)
+    @dishes = load_menu(menu_name)
+  end
+
+  def show
+    items = dishes.map { |item, price| "#{item}: £#{price}"}
+    "Menu\n\n#{items.join("\n")}"
+  end
+
+  def listed? item
+    dishes.include? item
+  end
+
+  private
+  def load_menu(menu_name)
+    file_name = "lib/menus/#{menu_name.to_s}.menu"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)